Shopify/hydrogen
# Add infinite scroll example

### HOW to test your changes?

1. Make sure to `npm install && npm run build` at the root.
2. Start the app: `cd examples/infinite-scroll && npm run dev`
3. Navigate to a collection: http://localhost:3000/collections/men
4. Scroll down the page, and make sure that new products automatically load.

#### Post-merge steps



#### Checklist
